nat-lab/bin/upnp-gw

```diff
@@ -7,7 +7,6 @@ upnpd eth0 eth1
 echo 1 > /proc/sys/net/ipv4/ip_forward
 
 # Acquire public IP address
-public_ip=$(ip route get 10.0.0.1 | egrep -o 'src [0-9.]+' | cut -f 2 -d ' ')
 public_itf=$(ip route get 10.0.0.1 | egrep -o 'dev [a-z0-9]+' | cut -f 2 -d ' ')
```

Comment:
Same as https://github.com/NordSecurity/libtelio/pull/96/files#r1324148575

Reply:
What's wrong with this one? Link is dead.

Reply:
It can also be simplified with `awk` :)

Reply:
Done